pkg/run/watch/secret.go

```diff
@@ -0,0 +1,39 @@
+package watch
+
+import (
+	"math/rand"
```

Comment:
We shouldn't use this package for security-sensitive applications.
```suggestion
	"crypto/rand"
```

> Package rand implements pseudo-random number generators unsuitable for security-sensitive work. ([source](https://pkg.go.dev/math/rand#pkg-overview))